plugins/GooglePlus/class.googleplus.plugin.php

```diff
@@ -246,7 +246,7 @@ public function setup() {
      */
     public function signInButton($type = 'button') {
         $target = Gdn::request()->post('Target', Gdn::request()->get('Target', url('', '/')));
-        $url = url('entry/googleplusauthredirect?Target='.$target);
+        $url = url('entry/googleplusauthredirect?Target='.$target, true);
```

Comment:
If the issue is a double call to url() why not just remove this call?

Reply:
I concur.